src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -336,7 +312,7 @@ pre.ignore:hover, .information:hover + pre.ignore {
 .tooltip::after {
 	background-color: #314559;
 	color: #c5c5c5;
-	border: 1px solid #5c6773;
+	border: 1px solid var(--border-color);
```

Comment:
I think we should no longer need to set the `border` property in stylesheets at all, right?

Reply:
You mean we could write:

```css
body {
    border-color: var(--border-color);
}
```

?

Reply:
No, I mean there should be a rule in rustdoc.css that says `.tooltip::after { border: 1px solid var(--border-color) }`, and this property should be deleted from the ayu theme.

Reply:
Oh sorry, I didn't pay attention to the file. You're right!

Reply:
Actually, `ayu` is a bit on its own here: the other themes don't have a border around the tooltip (you can see them in https://doc.rust-lang.org/nightly/std/string/struct.String.html#deref). A preference over what should be done?

Reply:
I think all the themes should have a thin border around the tooltip. This is consistent with what we do for the help and settings overlays, and for the notable traits overlay.

Reply:
Good point. I'll add the border color on all tooltips then.